### PR TITLE
Containerize browsertests

### PR DESCRIPTION
**Pull request by erkannt**
- testcafe container with our browsertests
- changes to ci compose, mocker conf to allow testcafe to point at `reviewer-client_nginx` instead of localhost
- pass base_url as envvar to browsertests <<< **browsertest crew please check if this is a sensible approach**
- fold image push into browsertest job as the image is over 700MB
- simplify github actions in passing

For `start_dev` and `start_test` use `test_browser`. This runs against localhost and doesn't use the browsertest container.

To run browsertest container: `setup`, `build_prod`, `start_ci`, `test_browser_containerized` or just `run_ci` (latter will also run unittests and lint).

### CODE REVIEW

File: tests/browser-tests/Login.browser.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { DashboardPage, FilesPage, LoginPage, AuthorDetailsPage, DetailsPage, NavigationPane } from '../page-objects';
 
-fixture`Getting Started`.page`http://localhost:9000`;
+const BASE_URL = process.env.BASE_URL || 'localhost:9000';
```
Comment (davidcmoulton) [inline tests/browser-tests/Login.browser.ts:3]: Just spit balling on the developer experience, but does it make sense to move this into a local module? Then you could have 
```JavaScript
import { BASE_URL } from './path/to/my-module'
```
Then the remember-to-always-define-the-constant turns into just another import. Might be easier to document, too.

Will defer to @Cooryd and @will-byrne who have the experience in this codebase as to whether this is a good idea or not.

Comment (erkannt) [reply tests/browser-tests/Login.browser.ts:3]: As this is blocking me right now, I'll merge but am happy to make another PR to implement something like this.

Comment (will-byrne) [reply tests/browser-tests/Login.browser.ts:3]: I like this suggestion

Comment (davidcmoulton) [reply tests/browser-tests/Login.browser.ts:3]: PR implementing this at https://github.com/libero/reviewer-client/pull/247

**Pull request merged**